posthog/permissions.py

```diff
@@ -19,7 +20,15 @@
 CREATE_METHODS = ["POST", "PUT"]
 
 
-def extract_organization(object: Model) -> Organization:
+def extract_organization(object: Model, view: View) -> Organization:
+    # This is set as part of the TeamAndOrgViewSetMixin to allow models that are not directly related to an organization
+    organization_id_rewrite = getattr(view, "filter_rewrite_rules", {}).get("organization_id")
+    if organization_id_rewrite:
+        for part in organization_id_rewrite.split("__"):
+            if part == "organization_id":
+                continue
```

Comment:
In this wouldn't `break` be the fully correct statement?

Reply:
Or fully fully correctly, we should check if the rewrite ends with `__organization_id`: if it does, remove it; if it doesn't, we shouldn't use the rewrite at all

Reply:
I guess it doesn't really matter (both work). 

For the other part - if you use the rewrite rule "organization_id": "foo" and it _doesn't_ resolve to an organization ID then its pretty borked anyways? 

Happy to make changes for clarity though